### DISCUSSION TITLE
Tree supports why solid infill?

### DISCUSSION THREAD

**Discussion by jzagaja** [OrcaSlicer/OrcaSlicer#5066] [General]
Why tree supports other then default have solid infill?

![WhatsApp Image 2024-04-15 at 14 03 26 (1)](https://github.com/SoftFever/OrcaSlicer/assets/71825386/0fed24a8-d6be-42a8-930a-1aac0c8cbe9f)

**Comment by hliebscher** [comment]
it works... you have wrong density of tree 
![tree](https://github.com/SoftFever/OrcaSlicer/assets/9274162/63a09d06-a4d6-445d-96da-108bc83362da)

**Comment by jzagaja** [comment]
Tree density? Don't see such an option.
